src/main/java/steps/AndroidLoginSteps.java

```diff
@@ -0,0 +1,56 @@
+package steps;
+
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.junit.jupiter.api.Assertions;
+import org.openqa.selenium.By;
+import skynet.runner.WebDriverManager;
+import skynet.utils.Logger;
+
+public class AndroidLoginSteps {
+    @When("user tap on the Login text button")
+    public void tapOnLoginButton() throws Exception {
+        WebDriverManager.getAppiumDriver().findElement(By.id("textViewLogin")).click();
+        Logger.info("should see correct Login text button");
+    }
+
+    @And("user select and tap on Germany")
+    public void selectAndTapOnDropDown() throws Exception {
+        Logger.info("click on the drop-down button");
+        WebDriverManager.getAppiumDriver().findElement(By.xpath("//android.widget.TextView[contains(@text,'Germany')]")).click();
+        Thread.sleep(2000);
```

Comment:
Please do not use `Thread.sleep`s. We should already be able to bypass this with the Waits if necessary.